ci/travis_install_conda.sh

```diff
@@ -23,13 +23,20 @@ fi
 wget -O miniconda.sh $MINICONDA_URL
 
 export MINICONDA=$HOME/miniconda
+export CONDA_PKGS_DIR=$HOME/.conda_packages
```

Comment:
This should actually be `CONDA_PKGS_DIRS`.  For conda, the value can be a single directory or a comma-delimited list.  Just read over the Travis CI docs on caching, and I think with that change, this configuration should work.